src/User.ts

```diff
@@ -124,4 +125,8 @@ export class User {
         Logger.createStatic("User", "fromStorageString");
         return new User(JSON.parse(storageString));
     }
+
+    public async dpopProof(url: string, httpMethod?: string): Promise<string> {
```

Comment:
not used at all -> remove

Reply:
Ok, but as pointed out on my original PR we need to expose the dpopProof somehow to the consumer so that it can be used as a request header for subsequent authenticated fetch requests to protected resources. Where would you suggest this be done? Here is an [example](https://github.com/dbfr3qs/react-oidc-context/blob/91f84e7118a3f7d5bdda4115536a79f0b858a587/example/index.tsx#L30-L38) using the react-oidc-client.

Reply:
> Ok, but as pointed out on my [original PR](https://github.com/authts/oidc-client-ts/pull/1361#discussion_r1465720125) we need to expose the generation of dpop proofs somehow to the consumer so that a proof can be used in the DPoP request header for subsequent authenticated fetch requests to protected resources. Where would you suggest this be done? Here is an [example](https://github.com/dbfr3qs/react-oidc-context/blob/91f84e7118a3f7d5bdda4115536a79f0b858a587/example/index.tsx#L30-L38) using the react-oidc-client

Ah i forgot that, makes totally sense.

I propose adding it with a describing comment to the `UserManager` class. This way `DPoPService` does not need to be static. The parameter to this function must probably be the user object, to have access to the access_token.

Reply:
Cool - that makes perfect sense 👍